### ISSUE TITLE
[Boomerang] Reminder email doesn't have codes

### ISSUE THREAD

**Issue by SorinaFlorean** [mozilla/fxa#2413]
Prerequisites: 
- Perform all steps to create a new account until you reach the 'Enter verification code' page.
- Wait for the "Reminder: Complete registration email".

Steps to reproduce;
1. Open the email and check the content.

Expected result:
A registration code is presented in order to confirm the account.

Actual result: 
"Confirm email" link is displayed.

**Comment by clouserw** [comment]
@davismtl will review

**Comment by shane-tomlinson** [comment]
A reminder email can't really have a registration code because the user would most likely not have anywhere to type it. Since the first reminder email is sent a week after the user signed up, the most likely scenario is that the user closed the original tab where it would be possible to enter a code. Showing the link within the email for verification reminds seems like the right thing to do.

**Comment by davismtl** [comment]
I agree that links will likely need to stick around, I just wanted to make sure we flushed out all the options.

Links seem like the obvious answer but are we forgetting any details? One of our goals with codes was to eliminate flows and reduce engineering complexity. Are we still achieving that if we keep links around in reminder emails?

With links in the reminder emails, how do we drive the person back into the right product? Could we mention the service they registered to on the page that opens following the link? Is there a way for us to do that with minimal complexity?

Somewhat related but not quite... In product, when the user is in an unverified state because they haven't confirmed their email, do we send them a link or a code? I think a link would make sense too but I'm just playing devil's advocate because as always the devil's in the details. 😈

**Comment by rfk** [comment]
> In product, when the user is in an unverified state because they haven't confirmed
> their email, do we send them a link or a code?

Another option here would be for the product code to open a new tab (or similar) that takes the user to the form where then can enter a code, which might reduce overall complexity.

**Comment by ryanfeeley** [comment]
Repeated from Slack…
<img width="885" alt="image" src="https://user-images.githubusercontent.com/68213/67043642-8aa2ee00-f0f8-11e9-85bc-2a04ddc25c4d.png">

**Comment by davismtl** [comment]
@ryanfeeley this works for the unverified state in the browser but wouldn't work for the reminder emails we automatically send after 2 and 5? days.

What might this look like for RPs?
